groupnews-service/src/main/java/com/epam/internship/UserService.java

```diff
@@ -8,4 +8,6 @@
 
 	public User createUser(String name, String email);
 	public List<User> getAllUsers();
+	public boolean emailAlreadyExists(String email);
```

Comment:
I get this error : Cannot reduce the visibility of the inherited method from UserService.
Also, should I make it private then or other modifier would be needed?